cannot run kotlin on Windows because wrong extension of kotlinc

**Describe the bug**
the code now will add `.exe` extension for `kotlinc`  in Windows, but the kotlin-compiler downloaded only contains `kotlinc` and `kotlinc.bat`. Therefore it should use `kotlinc.bat` in Windows, otherwise it causes exception.

**To Reproduce**
Steps to reproduce the behavior:
on Windows
1. `jbang init -t hello.kt hello.kt`
2. `jbang --verbose .\hello.kt`

**Expected behavior**
output Hello World
instead of 
```
[jbang] [ERROR] Cannot run program "C:\Users\...\.jbang\cache\kotlincs\1.6.10\kotlinc\bin\kotlinc.exe": CreateProcess error=2
```

**JBang version**
Paste output of `jbang version --verbose`
`[jbang] jbang version 0.97.0`

--- a/src/main/java/dev/jbang/net/KotlinManager.java
+++ b/src/main/java/dev/jbang/net/KotlinManager.java
@@ -19,10 +19,9 @@ public class KotlinManager {
 	public static String resolveInKotlinHome(String cmd, String requestedVersion) {
 		Path kotlinHome = getKotlin(requestedVersion);
 		if (kotlinHome != null) {
-			if (Util.isWindows()) {
-				cmd = cmd + ".bat";
-			}
-			return kotlinHome.resolve("bin").resolve(cmd).toAbsolutePath().toString();
+			Path dir = kotlinHome.toAbsolutePath().resolve("bin");
+			Path cmdPath = Util.searchPath(cmd, dir.toString());
+			return cmdPath != null ? cmdPath.toString() : cmd;
 		}
 		return cmd;
 	}

--- a/src/main/java/dev/jbang/net/KotlinManager.java
+++ b/src/main/java/dev/jbang/net/KotlinManager.java
@@ -20,7 +20,7 @@ public class KotlinManager {
 		Path kotlinHome = getKotlin(requestedVersion);
 		if (kotlinHome != null) {
 			if (Util.isWindows()) {
-				cmd = cmd + ".exe";
+				cmd = cmd + ".bat";
 			}
 			return kotlinHome.resolve("bin").resolve(cmd).toAbsolutePath().toString();
 		}

